core/src/main/java/hivemall/classifier/KernelizedPassiveAggressiveUDTF.java

```diff
@@ -0,0 +1,396 @@
+/*
+ * Hivemall: Hive scalable Machine Learning Library
+ *
+ * Copyright (C) 2015 Makoto YUI
+ * Copyright (C) 2013-2015 National Institute of Advanced Industrial Science and Technology (AIST)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package hivemall.classifier;
+
+import hivemall.common.LossFunctions;
+import hivemall.model.FeatureValue;
+import hivemall.model.PredictionResult;
+import hivemall.utils.collections.FloatArrayList;
+
+import java.util.ArrayList;
+import java.util.BitSet;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+import org.apache.hadoop.hive.ql.exec.UDFArgumentException;
+import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspector;
+import org.apache.hadoop.hive.serde2.objectinspector.StructObjectInspector;
+
+public class KernelizedPassiveAggressiveUDTF extends BinaryOnlineClassifierUDTF {
+
+    private float pkc;
+    private int degree;
+    private FloatArrayList alpha;
+    private List<FeatureValue[]> supportVectors;
```

Comment:
holding `supportVectors` are not feasible. So, better to remove the plain `Kernelized PassiveAggressive` implementation by making KernelizedPassiveAggressiveUDTF as an `abstract` class.

Provide only `PKI PA` and `Kernel Expansion PA`.